services/controllerhost/queueDepth.go

```diff
@@ -324,14 +325,20 @@ func (qdc *queueDepthCalculator) addExtentBacklog(
 
 func (qdc *queueDepthCalculator) computeBacklog(cgDesc *shared.ConsumerGroupDescription, cgExtent *metadata.ConsumerGroupExtent, storeMetadata *storeExtentMetadata, storeID string, logger bark.Logger) int64 {
 
-	var backlog int64
+	var backlog int64 = 0 // default
```

Comment:
I put that to make it explicit that we rely on a default of "0". I'll probably just leave a comment there that says this.